plugin/js/Util.js

```diff
@@ -45,9 +45,10 @@ var util = (function () {
         svg.setAttribute("version", "1.1");
         svg.setAttribute("preserveAspectRatio", "xMidYMid meet");
         svg.setAttribute("viewBox", "0 0 " + width + " " + height);
+        svg.setAttribute("data-origin", origin);
         let newImage = doc.createElementNS("http://www.w3.org/2000/svg","image");
         svg.appendChild(newImage);
-        newImage.setAttribute("xlink:href", href);
+        newImage.setAttribute("xlink:href", ".." + href.substr(5));
```

Comment:
replace "OEBPS" with "..". Only gets done when retrieving href for the html. Do not do this elsewhere or it will break a lot of things.